### PR TITLE
React Native 0.60 support (Cocoapods)

### PR DESCRIPTION
**Pull request by jonathanort**

### CODE REVIEW

**Comment by fylock** [comment]
Any chance we could get someone to approve and merge this PR?

File: RNAzureNotificationHub.podspec [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  s.name         = "RNAzureNotificationHub"
+  s.version      = package['version']
+  s.summary      = package['description']
+  s.license      = package['license']
+
+  s.authors      = package['author']
+  s.homepage     = package['homepage']
+
+  s.platform     = :ios, "7.0"
+  s.source       = { :git => "https://github.com/jonathanort/react-native-azurenotificationhub.git", :tag => "master" }
```
Comment (phongcao) [inline RNAzureNotificationHub.podspec]: Can you please update the source? I think it should be `https://github.com/CatalystCode/react-native-azurenotificationhub.git`.

**Comment by phongcao** [comment]
@jonathanort Thanks for your contribution. Can you please check my comment?

File: RNAzureNotificationHub.podspec [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  s.source_files  = "ios/**/*.{h,m}"
+  s.requires_arc = true
+  s.ios.dependency 'WindowsAzureMessaging'
```
Comment (phongcao) [inline RNAzureNotificationHub.podspec]: The readme file will need to be updated since `WindowsAzureMessaging` project isn't included anymore.

Comment (jonathanort) [reply RNAzureNotificationHub.podspec]: Thanks for the feedback, it should now be resolved

File: RNAzureNotificationHub.podspec (hunk 1 above)
Comment (jonathanort) [reply RNAzureNotificationHub.podspec]: Thanks, this should now be resolved

**Pull request merged**